about.html

```diff
@@ -0,0 +1,100 @@
+      </div>
+      <section class="mobile-menu">
+        <ul class="mobile-menu-background">
+          <li><a href="index.html">
+            <div class="logo-container">
+              <div class="gear"><span class="material-symbols-rounded">settings_suggest</span></div>
+              <div class="large-text">GhEng</div>
+            </div>
+          </a></li>
+          <li><a href="about.html" class="orange">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li class="campaign"><a href="#" class="orange">Campaign</a></li>
+        </ul>
+      </section>
+    </nav>
+  </header>
+  <!-- end common header files -->
+  <main>
+    <section class="headline">
+      <h1><span class="hello">"Hello Engineers!"</span><br><strong>ANNUAL GENERAL<br class="not-about"> CONFERENCE<br class="not-about"> 2024</strong></h1>
+      <hr class="indesktop">
+      <p class="intro">This conference will provide a common platform for all practitioners and affiliates from diverse engineering and related backgrounds to share ideas, experiences, and network.</p>
+      <p class="contact">Please contact us per Email for any further questions <br class="remove">about the annual Conference 2024!</p>
+      <P class="email"><strong>econference2024@ecaccra.org</strong> </P>
+    </section>
+    <section class="logo">
+      <h2>Engineer's Annual <br class="remove"> Conference 2024 Logo</h2>
+      <hr>
+      <p>The logo for the 2024 conference was decided <br>through the logo competion from 8. June to 7. July.</p>
+      <div class="big-container">
+        <div class="make-logo-container">
+          <div class="make-gear"><span class="material-symbols-rounded big-gear">settings_suggest</span></div>
+          <div class="make-large-text" >GhEng</div>
+        </div>
+      </div>
+    </section>
+    <section class="past-events">
+      <h2>See the past Conferences</h2>
+      <hr>
+      <p>Take a look at the last two conferences which took place <br class="remove">in Tarkorad and Sunyani.</p>
+      <div class="previous-container">
+        <div class="prev-conf1">
+          <h3>2021</h3>
+          <p>Annual Conference 2021 in Tarkoradi</p>
+        </div>
+        <div class="prev-conf2">
+          <h3>2019</h3>
+          <p>Annual Conference 2019 in Sunyani</p>
+        </div>
+      </div>
+    </section>
+    <section class="partner">
+      <h2>Partner</h2>
+      <hr>
+      <div class="companies">
+        <div><i class="fab fa-brands fa-airbnb"></i></div>
+        <div><i class="fab fa-brands fa-google"></i></div>
+        <div><i class="fab fa-brands fa-amazon"></i></div>
+        <div><i class="fa fa-solid fa-snowflake"></i></div>
+        <div><i class="fab fa-brands fa-stripe"></i></div>
+      </div>
+    </section>
+  </main>
+  <footer>
+    <p>&#169 2024 <br>Some Rights Reserved</p>
+  </footer>
```

Comment:
- Hey, please, the footer should contain your logo aligned to the left and text aligned to the right to match the design as much as you can. let's fix this. see the images below:


*design*
![Screenshot from 2023-03-16 19-05-03](https://user-images.githubusercontent.com/59397159/225712708-5ca37a3e-6e95-47fc-9b30-dd5e49815dfa.png)
*Your design*
![Screenshot from 2023-03-16 19-00-29](https://user-images.githubusercontent.com/59397159/225712714-ca961e38-c424-4ae0-99ba-7291ae892acf.png)